chain-signatures/node/src/backlog/mod.rs

```diff
@@ -200,7 +202,12 @@ impl Default for Backlog {
 
 impl Backlog {
     pub fn new() -> Self {
+        Self::persisted(CheckpointStorage::in_memory())
```

Comment:
Naming and logic around `inMemory`/`persisted` and `new()`/`persisted()` is a bit confusing. It is unclear if you recover old data or not and what is stored in memory. I would say `recover_from_storage` make more sense to me. But maybe I'm a bit opiniated here.